libs/core/langchain_core/load/serializable.py

```diff
@@ -107,6 +107,11 @@ class Serializable(BaseModel, ABC):
         as part of the serialized representation.
     """
 
+    # Remove default BaseModel init docstring.
+    def __init__(self, **kwargs: Any) -> None:
+        """"""
+        super().__init__(**kwargs)
```

Comment:
```suggestion
    def __init__(self, *args: Any, **kwargs: Any) -> None:
        """"""
        super().__init__(*args, **kwargs)
```
Think ya might need this